prediction_market_agent/agents/think_thoroughly_agent/deploy.py

```diff
@@ -14,6 +14,11 @@
 
 
 class DeployableThinkThoroughlyAgent(DeployableAgent):
+    model: str = "gpt-4-turbo-2024-04-09"
```

Comment:
Aren't costs too prohibitive? That's why I resorted back to gpt-3.5

Reply:
gpt-4-turbo is much better on costs than gpt-4 (that was enabled by default back then):

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/0b9ade73-ca06-4796-8590-a12f2d0fbfe6)

Unfortunately, gpt-3.5-turbo doesn't work well here. So I'm afraid this is the cheapest choice that (seems to) work.